cmake/common.cmake

```diff
@@ -88,6 +88,10 @@ option(SAN "Enable Address and Undefined Behavior Sanitizers" OFF)
 option(DISABLE_QUOTE_VERIFICATION "Disable quote verification" OFF)
 option(BUILD_END_TO_END_TESTS "Build end to end tests" ON)
 option(COVERAGE "Enable coverage mapping" OFF)
+option(AZURE_LOG_ANALYTICS "Enable azure analytics log collection" OFF)
```

Comment:
I wonder if this should be an environment variable rather another cmake flag as it only affects `ctest`. We could then define the env var in our Azure Pipeline.